cglm.cmake.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix="${prefix}"
+libdir="${exec_prefix}/lib"
+includedir="${prefix}/include"
```

Comment:
```suggestion
prefix=@CMAKE_INSTALL_FULL_PREFIX@
exec_prefix="${prefix}"
libdir=@CMAKE_INSTALL_FULL_LIBDIR@
includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
```

Reply:
So does simply reverting the original broken commit, surely?

Reply:
https://github.com/recp/cglm/pull/272